src/accel/SharedParams.cc

```diff
@@ -19,6 +19,10 @@
 #include <G4RunManager.hh>
 #include <G4Threading.hh>
 
+#ifdef _OPENMP
+#    include <omp.h>
```

Comment:
```suggestion
#if CELERITAS_USE_OPENMP
```

Reply:
Probably need to add the openmp cmake target.